docs/reference/commandline/dockerd.md

```diff
@@ -3,6 +3,8 @@ title: "dockerd"
 aliases: ["/engine/reference/commandline/daemon/"]
```

Comment:
I think this one has been removed on the master branch 😅

Reply:
Good catch; guess I forgot to update it after I did on master. Updating now